src/main/java/appeng/client/gui/implementations/GuiMEMonitorable.java

```diff
@@ -470,18 +478,40 @@ protected void repositionSlot( final AppEngSlot s )
 	@Override
 	protected void keyTyped( final char character, final int key ) throws IOException
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The Cyclomatic Complexity of this method "keyTyped" is 12 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)